.github/workflows/release.yaml

```diff
@@ -0,0 +1,215 @@
+# From https://github.com/BurntSushi/ripgrep/blob/master/.github/workflows/release.yml
+
+name: release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+# We need this to be able to create releases.
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: create-release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get the release version from the tag
+        if: env.VERSION == ''
+        run: echo "${{ github.ref_name }}" | sed -En 's/^(.*)$/VERSION=\1/p' >> $GITHUB_ENV
+
+      - name: Get the gvltctl release version from the tag
+        if: env.RELEASE_NAME == ''
+        run: echo "RELEASE_NAME=${{ github.ref_name }}" >> $GITHUB_ENV
+      - name: Show the version
+        run: |
+          echo "version is: $VERSION"
+
+      - name: Check that tag version and Cargo.toml version are the same
+        shell: bash
+        run: |
+          if ! grep -q "version = \"$VERSION\"" Cargo.toml; then
+            echo "version does not match Cargo.toml" >&2
+            exit 1
+          fi
+
+      - name: Create GitHub release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh release create v$VERSION --draft --title v$VERSION
+    outputs:
+      version: ${{ env.VERSION }}
+      release_name: ${{ env.RELEASE_NAME }}
+
+  build-release:
+    name: build-release
+    needs: ['create-release']
+    runs-on: ${{ matrix.os }}
+    env:
+      # For some builds, we use cross to test on 32-bit and big-endian
+      # systems.
+      CARGO: cargo
+      # When CARGO is set to CROSS, this is set to `--target matrix.target`.
+      TARGET_FLAGS:
+      # When CARGO is set to CROSS, TARGET_DIR includes matrix.target.
+      TARGET_DIR: ./target
+      # Bump this as appropriate. We pin to a version to make sure CI
+      # continues to work as cross releases in the past have broken things
+      # in subtle ways.
+      CROSS_VERSION: v0.2.5
+      # Emit backtraces on panics.
+      RUST_BACKTRACE: 1
+      # Build static releases with PCRE2.
+      PCRE2_SYS_STATIC: 1
+      # To ensure openssl is static linked
+      OPENSSL_STATIC: 1
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - build: stable-x86
+          os: ubuntu-latest
+          rust: stable
+          target: x86_64-unknown-linux-gnu
+          qemu: i386
+#       - build: stable-aarch64
```

Comment:
Why is it commented out?

Reply:
To enable it later if we want those to be able to compile